### ISSUE TITLE
Breaking syntax highlight in other languages

### ISSUE THREAD

**Issue by corderop** [vscode-django/vscode-django#101]
After installing the extension, Vue and some JS files do not have syntax highlights. It shows these files as `django-txt` language files.

**Without the extension**:
![imagen](https://user-images.githubusercontent.com/31637109/156561871-bcdde656-2370-4996-81cc-6deaa7c22ec2.png)

**With the extension**:
![imagen](https://user-images.githubusercontent.com/31637109/156561994-0c9391bc-f854-42c3-9c6f-e4b122bd29f2.png)

**Comment by batisteo** [comment]
Are these files within a `templates` directory?

**Comment by corderop** [comment]
> Are these files within a `templates` directory?

Yes, this could be the problem. It's working good with files outside the `template` folder

**Comment by batisteo** [comment]
The part responsible for this behaviour is https://github.com/vscode-django/vscode-django/blob/1.10/package.json#L96-L107
But I’m not sure how to deal with it.

**Comment by corderop** [comment]
I'm not much into Django used with templates. I suppose you have already overcome this solution but maybe only support email templates with certain extensions?

**Comment by franciscocobas** [comment]
This happened to me as well! I disabled the extension and problem fixed 😛
